graphql-server/migrations/1589054544959_update_permission_player_public_table_games/up.yaml

```diff
@@ -0,0 +1,28 @@
+- args:
+    role: player
+    table:
+      name: games
+      schema: public
+  type: drop_update_permission
+- args:
+    permission:
```

Comment:
Very cool that these get auto-generated, but is this preferable? There were a handful of permission updates which resulted in a few migration files that could probably be condensed.

Reply:
Coming very soon :p

https://github.com/avimoondra/fishbowl/pull/29 

So for now, don't worry how many migrations get created, it's normal to generate a bunch. Eventually there will be less, and easier to review (by diffing migrations/metadata.yaml in review)